### PR TITLE
Add Terraform & kubectl extension to devcontainer

### PR DESCRIPTION
**Pull request by ianychoi**
## Purpose

This requires for session02 to be completed.

Reference link: https://github.com/devcontainers/features

## Does this introduce a breaking change?
[x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

[x] Feature

## How to Test
*  (Execute "terraform" with the new devcontainer environment
*  (Execute "kubectl" with the new devcontainer environment

### CODE REVIEW

File: .devcontainer/post-create.sh [hunk 1]
```diff
@@ -90,3 +90,22 @@ curl https://raw.githubusercontent.com/justinyoo/devcontainers-dotnet/main/oh-my
...
+done
+echo -n "$LATEST"
+}
+
+TERRAFORM_VERSION=$(getTerraformLatestVersion)
+
+cd ~
+wget "https://releases.hashicorp.com/terraform/"$TERRAFORM_VERSION"/terraform_"$TERRAFORM_VERSION"_linux_amd64.zip"
+unzip "terraform_"$TERRAFORM_VERSION"_linux_amd64.zip"
+sudo install terraform /usr/local/bin/
```
Comment (justinyoo) [inline .devcontainer/post-create.sh:111]: 혹시 이것 대신에 우리 코드스페이스의 공식 피처인 이걸 쓰는 것이 좋을 듯 싶습니다

https://github.com/devcontainers/features/tree/main/src/terraform

**Review by justinyoo — CHANGES_REQUESTED** [review]
`post-create.sh` 파일을 통해 테라폼 CLI 설치하는 대신, GitHub 코파일럿의 devcontainer feature 기능을 이용하는 것으로 수정해 주시면 좋겠습니다!

**Comment by ianychoi** [comment]
@justinyoo  Thank you for your comment - I reflected + added `kubectl` related feature as one more commit for this PR.

**Review by justinyoo — APPROVED** [review]
LGTM

**Pull request merged**